docs/can-guides/contribute/site-style-guide.md

```diff
@@ -121,3 +121,15 @@ npm install can --save
 
 The following is a very long sentence that will hopefully go across many lines because it
 is so long and filled with <code>code elements</code>, <strong>bold elements</strong>, <em>italic elements</em>, <a href="#">link elements</a>.
+
+## Screenshots
+
+Use the `bit-docs-screenshot` class on images to center them. Add a `width` to the image to set a max width.
+
+### With width
+
+<img alt="" class="bit-docs-screenshot" src="https://canjs.com/docs/can-guides/images/devtools/panel-component-selected.png" width="600px" />
```

Comment:
wouldn't you set `max-width`?

Reply:
No, I don’t think so (and I don’t think that attribute exists in html). Ideally we would have height and width set for all images. The CSS sets the `max-width` to 100% so it’s dynamic.